query-algebrizer/src/cc.rs

```diff
@@ -69,7 +69,7 @@ pub enum DatomsColumn {
 
 impl DatomsColumn {
     pub fn as_str(&self) -> &'static str {
-        use DatomsColumn::*;
+        use DatomsColumn::{Attribute, Entity, Tx, Value, ValueTypeTag};
```

Comment:
Note that this also imports all inner types of a sum type, so there's zero change to break "unowned" code.